apps/backoffice-v2/.env.example

```diff
@@ -1,4 +1,4 @@
-VITE_API_URL=http://localhost:3000/api/v1/internal
+VITE_API_URL=api/v1/internal
```

Comment:
It won't work on local environment since the API server is on port 3001

Reply:
Ok, anything you can recommend here then ?